05-Organizing-and-Executing-Simulations/01-Organizing-Data.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<p>Something to explain that the project is the sum of the state points, the simulations carried out at these state points and the resulting output produced? Also that the location of the resulting <code>signac.rc</code>, the project root directory are both defined by where <code>signac.init_project()</code> is called (it may not be obvious that you can't just call this anywhere and that the project isn't actually defined by the name argument)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done.</p>